ui/lib/core/addon/components/policy-example.hbs

```diff
@@ -0,0 +1,43 @@
+<div class="has-bottom-margin-s">
+  {{#if (eq @policyType "acl")}}
+    <p data-test-example-modal-text-acl>
+      ACL Policies are written in Hashicorp Configuration Language (
+      <ExternalLink @href="https://github.com/hashicorp/hcl">HCL</ExternalLink>
+      ) or JSON and describe which paths in Vault a user or machine is allowed to access. Here is an example policy:
+    </p>
+  {{else}}
+    <p class="has-bottom-margin-s" data-test-example-modal-text-rgp>
+      Role Governing Policies (RGPs) are tied to client tokens or identities which is similar to
+      <DocLink @path="/vault/tutorials/policies/policies">ACL policies</DocLink>. They use
+      <DocLink @path="/vault/docs/enterprise/sentinel">Sentinel</DocLink>
+      as a language framework to enable fine-grained policy decisions.
+    </p>
+    <p>
+      Here is an example policy that uses RGP to restrict access to the
+      <code class="tag is-marginless is-paddingless">admin</code>
+      policy such that a user named James or has the
+      <code class="tag is-marginless is-paddingless">Team Lead</code>
+      role can manage the
+      <code class="tag is-marginless is-paddingless">admin</code>
+      policy:
+    </p>
+  {{/if}}
+</div>
+<JsonEditor @value={{this.policyTemplate}} @mode="ruby" @readOnly={{true}} @showToolbar={{true}} />
+<div class="has-bottom-margin-m has-top-padding-s">
+  <p>
+    More information about
+    {{uppercase @policyType}}
+    policies can be found
+    <DocLink
+      @path={{if
```

Comment:
this conditional will have to chance when we add the third policy type. Another option could be to make this an object similar to what we did with `policyTemplates` so we return the correct link based on the policy type